README.md

```diff
@@ -26,35 +26,120 @@ flutter pub add flame_behaviors
 
 ## Usage ✨
 
-### Behavior
+### Entity
+
+The entity is the building block of a game. It represents a visual game object that can hold 
+multiple `Behavior`s which in turn define how the entity behaves.
+
```

Comment:
```suggestion
The entity is the building block of a game. It represents a visual game object that can hold multiple `Behavior`s, which in turn define how the entity behaves.

```

What does building block of a game mean? Does a game consist of multiple entities? I think it would be super helpful to zoom out and talk about the overall structure of a game (e.g. is game loop at all relevant here??) and how entities fit within that.